decidim-core/app/packs/src/decidim/redesigned_index.js

```diff
@@ -188,3 +203,15 @@ if ("Turbo" in window) {
   // document.addEventListener("DOMContentLoaded", () => {
   $(() => initializer());
 }
+
+// eslint-disable-next-line camelcase
+StreamActions.open_drawer = function() {
```

Comment:
@Crashillo should this be moved inside the initializer?

Reply:
No way. This is a feature given to the StreamActions object (in order to use `turbo_stream_action_tag`). Perhaps it could be isolated into another different file and imported in the index, but never inside the initializer